packages/muelu/src/Graph/Containers/MueLu_LWGraph_def.hpp

```diff
@@ -55,7 +55,7 @@ namespace MueLu {
   //void describe(Teuchos::FancyOStream &out, const VerbLevel verbLevel = Default) const {
   template <class LocalOrdinal, class GlobalOrdinal, class Node>
   void LWGraph<LocalOrdinal, GlobalOrdinal, Node>::print(Teuchos::FancyOStream &out, const VerbLevel verbLevel) const {
-    MUELU_DESCRIBE;
+    //    MUELU_DESCRIBE;
```

Comment:
No idea what that used to do, but why no more?

Reply:
That generates a FancyOStream (on rank 0 I think).  We don't want that, as we're passing a specific FOS in.